cub/benchmarks/bench/adjacent_difference/subtract_left.cu

```diff
@@ -52,8 +52,8 @@ struct policy_hub_t
 template <class T, class OffsetT>
 void left(nvbench::state& state, nvbench::type_list<T, OffsetT>)
 {
-  constexpr bool may_alias = false;
-  constexpr bool read_left = true;
+  constexpr cub::MayAlias may_alias   = cub::MayAlias::No;
+  constexpr cub::ReadOption read_left = cub::ReadOption::Left;
```

Comment:
Suggestion: inline those into the instantiation of `DispatchAdjacentDifference` below.